posix.mak

```diff
@@ -42,6 +42,9 @@ STABLE_DMD_CONF=$(STABLE_DMD).conf
 STABLE_RDMD=$(STABLE_DMD_ROOT)/dmd2/$(OS)/$(if $(filter $(OS),osx),bin,bin$(MODEL))/rdmd \
 	--compiler=$(STABLE_DMD) -conf=$(STABLE_DMD_CONF)
 
+# whether to use a special theme for nightly changelog builds
+BUILD_NIGHTLY=1
```

Comment:
The idea was that Martin can call the Makefile with `BUILD_NIGHTLY=0` and then it will generate the to be released page without the nightly texts. In other words:
- current date instead of "to be released"
- `VERSION` instead of `NIGHTLY_VERSION`
- no `(upcoming)` in the title
- ...